packages/block-library/src/columns/edit.js

```diff
@@ -72,6 +72,7 @@ function ColumnsEditContainer( {
 	} );
 	const innerBlocksProps = useInnerBlocksProps( blockProps, {
 		allowedBlocks: ALLOWED_BLOCKS,
+		allowedChildBlocks,
```

Comment:
I think this line can be removed, `allowedChildBlocks` isn't a recognized property for inner blocks:
https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/inner-blocks/README.md#props

Reply:
Yes, I agree that this line should be removed.